examples/pytorch/rgcn/fake_data_generator.py

```diff
@@ -0,0 +1,201 @@
+import pandas as pd
+import numpy as np
+import torch as th
+import dgl
+
+class CreateFakeData:
```

Comment:
Can we briefly describe the structure of the faked data?

Reply:
Done